src/main/java/uk/gov/digital/ho/pttg/application/HmrcClient.java

```diff
@@ -234,39 +239,52 @@ private String stripPlaceholderQueryParams(String href) {
     private String getIndividualLink(Individual individual, String accessToken, String matchUrl) {
         log.info("POST to {}", matchUrl);
 
-        Resource<String> resource = restTemplate.exchange(URI.create(matchUrl), HttpMethod.POST, createEntity(individual, accessToken), linksResourceTypeRef).getBody();
+        Resource<String> resource = null;
+
+        try {
+            resource = restTemplate.exchange(URI.create(matchUrl), POST, createEntity(individual, accessToken), linksResourceTypeRef).getBody();
+        } catch (RestClientResponseException e) {
```

Comment:
I noticed other people seemed to be working off HttpClientErrorException within this project, which is nice because it is only thrown for 4XX status codes.  However you may have a better reason for RestClientResponseException?